app/views/users/_metadata.html.erb

```diff
@@ -11,21 +11,5 @@
         </p>
       </div>
     <% end %>
-    <% if @user.employment_title.present? %>
```

Comment:
The `work` field automatically gets picked up by `@profile.ui_attributes_for(area: :header)` so we can remove this special handling.